extra/firefox/build

```diff
@@ -76,6 +76,7 @@ ac_add_options --disable-dbus
 ac_add_options --disable-gold
 ac_add_options --disable-tests
 ac_add_options --disable-vtune
+ac_add_options --disable-webrtc
```

Comment:
Would you mind adding a comment in the build file that `--disable-webrtc` is needed due to conflicts with `whatever.patch`, so people in the future know this is less about disabling bloat and more a limitation of firefox? (And maybe someone can take a stab at dealing with that).

Reply:
This should be fixed upstream shortly. Will add a comment for now however. https://bugzilla.mozilla.org/show_bug.cgi?id=1661450

Thanks